src/themes/dark.ts

```diff
@@ -14,6 +14,7 @@ export default {
   card: {
     backgroundColor: colors.neutral800,
     textColor: colors.white,
+    border: `0.063em solid ${  colors.darkGray}`,
```

Comment:
Why not use one of the neutral colors here?

`neutral50` through `neutral900` provide a range of grays that should be able to fit a border color instead of creating a new color.